src/gobupload/relate/update.py

```diff
@@ -242,10 +243,11 @@ def __init__(self, src_catalog_name, src_collection_name, src_field_name):
 
         # begin_geldigheid tmp table names
         datestr = datetime.now().strftime('%Y%m%d')
-        src_intv_tmp_table_name = f"tmp_{self.src_catalog_name}_{self.src_collection['abbreviation']}_intv_" \
-                                  f"{datestr}_{str(random.randint(0, 1000)).zfill(4)}".lower()
-        dst_intv_tmp_table_name = f"tmp_{self.dst_catalog_name}_{self.dst_collection['abbreviation']}_intv_" \
-                                  f"{datestr}_{str(random.randint(0, 1000)).zfill(4)}".lower()
+        characters = string.ascii_lowercase + ''.join([str(i) for i in range(10)])
+        src_intv_tmp_table_name = f"tmp_{self.src_catalog_name}_{self.src_collection['abbreviation']}_intv_{datestr}" \
+                                  f"_{''.join([random.choice(characters) for _ in range(6)])}".lower()
+        dst_intv_tmp_table_name = f"tmp_{self.dst_catalog_name}_{self.dst_collection['abbreviation']}_intv_{datestr}" \
+                                  f"_{''.join([random.choice(characters) for _ in range(6)])}".lower()
 
```

Comment:
Misschien hier een aparte methode van maken? Iets zoals random_string(length=6) of zo?